src/angular/datepicker/datepicker/datepicker.scss

```
@@ -3,15 +3,22 @@
 .sbb-datepicker {
   display: inline-block;
   position: relative;
-  width: 100%;
   color: var(--sbb-color-weak-accent);
+
+  .sbb-form-field-input-wrapper > & {
```

A little nit picking, but i think form-field-input-wrapper is more like an implementation detail of form field. @kyubisation your thoughts?
```suggestion
  .sbb-form-field > & {
```

---

I like it since it's more exlicit. I committed it without the `>` since `sbb-form-field` is not a direct parent.

---

I agree. However, the `>` operator should be removed in that case.